types/caverphone/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for caverphone 1.0.0
+// Project: https://github.com/tcort/caverphone
+
+
+/*
+* caverphone is a speech algorithm that generates a code representing the pronunciation of a word for fuzzy matching.
+*
+* @param { string } word -Start with a word
+* @return { string }
```

Comment:
FYI: these type hints are not needed in TypeScript files

```suggestion
* @param word -Start with a word
```